src/UI/templates/js/Modal/modal.js

```diff
@@ -25,15 +29,30 @@ il.UI = il.UI || {};
                 var $modal = $('#' + id);
                 $modal.modal(options);
             }
-        };
+			initializedModalboxes[signalData.id] = id;
+		};
 
         var closeModal = function (id) {
             $('#' + id).modal('close');
         };
 
+        /**
+         * Replace the content of the modalbox showed by the given showSignal with the data returned by the URL
+         * set in the signal options.
+         *
+         * @param id component ID
+         * @param signalData Object containing all data from the replace signal
+         */
+        var replaceFromSignal = function (id, signalData) {
+            var url = signalData.options.url;
+
+            il.UI.core.replaceContent(id, url, "component");
```

Comment:
Maybe il.UI.core.replace.component(id,url); 

would be a bit more telling of what is going on and less error prone.